docs/documentation.md

```diff
@@ -134,14 +130,14 @@ We also expose a few options that can be used to configure the default and initi
 
 <div class="codebox-combo">
 
-To build your search result page, you need to combine widgets together. Widgets are simple objects which hold the handler to part of the instant search lifecycle. Start by adding a `searchBox` widget, a `hits` widget and a `pagination` widget to build a fundamental results page.
+The build your search results page, you need to combine widgets together. Widgets are simple objects which hold the handler to part of the instant search lifecycle. Start by adding a `searchBox` widget, a `hits` widget and a `pagination` widget to build a fundamental results page.
 
 <div class="code-box">
-  <div class="code-sample-snippet js-toggle-snippet ignore">
+  <div class="code-sample-snippet ignore">
 {% highlight html %}
 <div id="search-box"></div>
 
-<script>
+<script type="text/javascript">

```

Comment:
I removed that in a previous PR, because it was unneeded (this is the default value). I'm sure there was an over-zealous conflict resolution here :)